generative_data_prep/data_prep/pipeline.py

```diff
@@ -16,6 +16,7 @@
 Data preparation pipeline for converting a jsonl file to tokenized hdf5 files consumable by SambaSuite.
 """
 
+import concurrent.futures
```

Comment:
Remove the multiprocessing Pool import below, and also probably remove this from the requirements since its not necessary any more

Reply:
I am not sure how you were allowed to commit this code with an unused import... ?

Are you running the pre-commit hook checks?

Reply:
maybe they aren't running, are they not enabled by default?

Reply:
oh I didn't install pre-commit

Reply:
hmmm just installed pre-commit, doesn't seem like it ran anything tho :(

Reply:
If this is working now you can resolve this!